text/0046-ttfd-automatic-transaction-span.md

```diff
@@ -0,0 +1,90 @@
+- Start Date: 2022-12-16
+- RFC Type: feature
+- RFC PR: https://github.com/getsentry/rfcs/pull/46
+- RFC Status: draft
+  
+# Summary. 
```

Comment:
Is that going to be a transaction, a span or a measurement?
How is this going to play along with https://docs.sentry.io/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation ?
Do we intend to do that for Fragments as well? https://docs.sentry.io/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-fragment-instrumentation

Reply:
The idea is to create a new span to attach to the automatic transactions of activities.
We could also later add the same for fragment transactions as well

Reply:
Mobile vitals is based on measurements, don't we intend to leverage our mobile vitals with this?
What if the transaction auto finishes before the user called `Sentry.reportFullyDrawn`?

Reply:
I feel like we could have both, right? The reason for spans is that we can detect performance issues server-side (which works based on spans). But having it as a measurement wouldn't hurt either?